packages/api-reference/src/components/Content/ReferenceEndpoint/RequestBody.vue

```diff
@@ -1,21 +1,39 @@
 <script setup lang="ts">
+import { ref } from 'vue'
+
 import type { RequestBody } from '../../../types'
 import Schema from '../Schema.vue'
 
-defineProps<{ requestBody?: RequestBody }>()
+const prop = defineProps<{ requestBody?: RequestBody }>()
+const selectedContentType = ref(
+  prop?.requestBody?.content
+    ? Object.keys(prop?.requestBody?.content)[0]
+    : undefined,
+)
 </script>
 <template>
-  <div v-if="requestBody && requestBody.content?.['application/json']">
+  <div v-if="prop?.requestBody">
     <div class="request-body-title">
       <slot name="title" />
+      <select
+        v-if="prop?.requestBody"
+        v-model="selectedContentType"
+        style="display: block">
+        <option
+          v-for="(value, key) in prop.requestBody?.content"
+          :key="key"
+          :value="key">
+          {{ key }}
+        </option>
+      </select>
     </div>
     <div
-      v-if="requestBody.content?.['application/json']"
+      v-if="prop?.requestBody.content?.[selectedContentType]"
```

Comment:
this is causing a build error for undefined being used as an index, we should change the line to this for an undefined check on `selectedContentType`

```
v-if="selectedContentType && prop?.requestBody.content?.[selectedContentType]"
```

```
12:31:40 AM: @scalar/api-reference:build: src/components/Content/ReferenceEndpoint/RequestBody.vue(31,41): error TS2538: Type "undefined" cannot be used as an index type.
12:31:40 AM: @scalar/api-reference:build: src/components/Content/ReferenceEndpoint/RequestBody.vue(36,43): error TS2538: Type "undefined" cannot be used as an index type.
```